mlflow/ml-package-versions.yml

```
@@ -242,6 +242,17 @@ spacy:
     run: |
       pytest tests/spacy/test_spacy_model_export.py --large
 
+spark-3.x:
+  package_info:
+    pip_release: "pyspark"
+
+  autologging:
+    minimum: "3.0.0"
+    maximum: "3.1.1"
+    run: |
+      pushd mlflow/java/spark && mvn package -DskipTests -q && popd
```

```suggestion
      pushd mlflow/java/spark && mvn package -DskipTests -q && popd
```

Can we remove this line for now?

---

This line is required because currently we use `log_models` method imported from `mflow.spark` module and it require init `mflow.spark` module

---

I think `mlflow.spark.log_model` can be used without `mlflow/java`. `mlflow.spark.log_model` requires `mlflow/java` only when the `sample_input` argument is supplied:

https://github.com/mlflow/mlflow/blob/77e5c44d483a6ca19bdb07165c32856928682bbf/mlflow/spark.py#L14-L20